### DISCUSSION TITLE
Norms when subsetting 10X multiome dataset using WNN vignette

### DISCUSSION THREAD

**Discussion by sylestiel** [satijalab/seurat#6729] [General]
Hi,

 Does subsetting of Seurat object  when using WNN analysis of 10x Multiome, RNA + ATAC vignette **require a repeat of SCT and LSI steps as well as the FindMultimodalNeighbors?**

Thanks!

**Comment by kathbosc** [comment]
Hi all, I have a similar question.. What is the best practice after subsetting an integrated multiome dataset (RNA and ATAC) before running WNN again? Any recommendation is greatly appreciated! Thanks a lot in advance!
